auth-web/src/services/business.services.ts

```diff
@@ -202,41 +202,18 @@ export default class BusinessService {
     })
   }
 
-  // *** TODO: this should return type "array of reviews"
-  static async fetchContinuationReviews (): Promise<any> {
-    // Mock data simulating the expected response structure from the API
-    await new Promise(resolve => setTimeout(resolve, 1000))
-    return new Promise(resolve => resolve({
-      data: [
-        {
-          reviewId: 1,
-          date: 'July 3, 2024',
-          nrNumber: 'NR 0001234',
-          businessIdentifier: 'LN958001',
-          completingParty: 'John Doe',
-          status: 'Approved'
-        },
-        {
-          reviewId: 2,
-          date: 'April 4, 2024',
-          nrNumber: 'NR 0001235',
-          businessIdentifier: 'LN786002',
-          completingParty: 'Jane Smith',
-          status: 'Resubmitted'
-        },
-        {
-          reviewId: 3,
-          date: 'June 15, 2024',
-          nrNumber: 'NR 0001239',
-          businessIdentifier: 'LN965002',
-          completingParty: 'Test Mark',
-          status: 'Awaiting Review'
-        }
-      ]
-    } as any))
+  static async fetchContinuationReviews (): Promise<ContinuationReviewIF[]> {
+    const url = `${ConfigHelper.getLegalAPIV2Url()}/admin/reviews`
+    const response = await axios.get(url)
+    const data = response.data.reviews
+    return data
+  }
```

Comment:
This will throw an error if response or response.data are falsy (null/undefined).

I recommend structuring this as on line 129 above.

Also, what happens if line 207 throws an error (eg, 403 or 404 or 5xx)? Should you return null from this method, or perhaps `[]`? Does the table support this?

Reply:
This function is obsolete now, right? If so then I recommend deleting it.

(And I see you handle my comments above in the other function: 👍 )